src/layout.js

```diff
@@ -25,7 +25,13 @@ export function render({ callbacks, options, random, selection, words }) {
 		onWordMouseOver,
 		onWordMouseOut,
 	} = callbacks;
-	const { colors, enableTooltip, fontStyle, fontWeight } = options;
+	const {
+		colors,
+		enableTooltip,
+		tooltipOptions,
```

Comment:
after line 33.  Since everything is sorted by convention so far, let's keep at it :)

Reply:
Cool, good catch.  Done and moved down.